pkg/apis/core/types.go

```diff
@@ -2571,6 +2571,12 @@ type PodSpec struct {
 	// More info: https://github.com/kubernetes/community/blob/master/keps/sig-network/0007-pod-ready%2B%2B.md
 	// +optional
 	ReadinessGates []PodReadinessGate
+	// RuntimeClassName refers to a RuntimeClass object with the same name, which should be used to
```

Comment:
Can we be more precise about which apigroup this type exists in?

Reply:
Done